### DISCUSSION TITLE
Can I programatically select an option from the data or initialize with an item already selected?

### DISCUSSION THREAD

**Discussion by jchew** [TarekRaafat/autoComplete.js#300] [Q&A]
Hello,

I am trying to get the autoComplete to initialize with a value from the data set already selected based on a query string parameter in the URL. If that is not possible, I could also set the value programatically after the autoComplete initializes, but I am having trouble achieving either.

I am using an Array of Objects as the data.

If anyone could help point me in the right direction, I would really appreciate it. Thanks!

**Comment by folknor** [comment]
https://github.com/TarekRaafat/autoComplete.js/issues/241 should have all the code you need.
